plugins/catalog-backend-module-bitbucket-cloud/config.d.ts

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2022 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+interface BitbucketCloudProviderConfig {
```

Comment:
may want to inline this below even if it's repeated - not sure it's good to have a separate type visible here. @Rugvip ? I think we had some problem with this at some point right

Reply:
Yep, please keep it inlined. Type names are global af so it's best to avoid them and put it all in `Config`.

As a side note I feel we should clearly add some linting against it, or flat out refuse separate types

Reply:
ok, I see. FYI: We have this at a few other places already, too.

Reply:
fixed